src/index.js

```diff
@@ -3,6 +3,7 @@ import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 import queryString from 'query-string';
 import { hot } from 'react-hot-loader';
+import { getFirstFocusable } from '@folio/stripes-components/util/getFocusableElements';
```

Comment:
Can we export `getFirstFocusable` from stripes-components so we don't have to reach in and get it by private path here?